### PR TITLE
blip and decidiffusion updates

### PR DESCRIPTION
**Pull request by vadthyavath**
- deci-decidiffusion-v1-0
- salesforce-blip2-opt-2-7b-vqa
- salesforce-blip2-opt-2-7b-image-to-text
- salesforce-blip-vqa-base
- salesforce-blip-image-captioning-base

### CODE REVIEW

File: assets/models/system/deci-decidiffusion-v1-0/description.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
-__Deci/DeciDiffusion-v1-0__ DeciDiffusion 1.0 is a diffusion-based text-to-image generation model. While it maintains foundational architecture elements from Stable Diffusion, such as the Variational Autoencoder (VAE) and CLIP's pre-trained Text Encoder, DeciDiffusion introduces significant enhancements. The primary innovation is the substitution of U-Net with the more efficient U-Net-NAS, a design pioneered by Deci. This novel component streamlines the model by reducing the number of parameters, leading to superior computational efficiency.
+`DeciDiffusion` 1.0 is an 820 million parameter latent diffusion model designed for text-to-image conversion. Trained initially on the LAION-v2 dataset and fine-tuned on the LAION-ART dataset, the model's training involved advanced techniques to improve speed, training performance, and achieve superior inference quality.
 
+For more details, review the <a href='https://deci.ai/blog/decidiffusion-1-0-3x-faster-than-stable-diffusion-same-quality/' target='_blank'>blog</a>.
```
Comment (gauravrajguru) [inline assets/models/system/deci-decidiffusion-v1-0/description.md:3]: are you adding more sections from this card?

File: assets/models/system/deci-decidiffusion-v1-0/description.md [hunk 2]
```diff
@@ -1,26 +1,58 @@
...
+
+2. The model was trained for 870k steps at a higher resolution of 512x512 on the same dataset to capture more fine-detailed information.
+
+3. Training for 65k steps with EMA, a different learning rate scheduler, and more qualitative data.
+
+4. Then the model underwent fine-tuning on a 2 million sample subset of the LAION-ART dataset.
+
+In phase 1, 8 X 8 X A100 GPUs, AdamW optimizer had been used with batch size 8192 and learning rate 1e-4. In phases 2-4, 8 X 8 X H100 GPUs, LAMB optimizer had been used with batch size 6144 and learning rate 5e-3.
+
+# Evaluation Results
```
Comment (gauravrajguru) [inline assets/models/system/deci-decidiffusion-v1-0/description.md:23]: If it's blank, better remove it.

Comment (vadthyavath) [reply assets/models/system/deci-decidiffusion-v1-0/description.md]: removed it!

File: assets/models/system/deci-decidiffusion-v1-0/description.md (hunk 1 above)
Comment (vadthyavath) [reply assets/models/system/deci-decidiffusion-v1-0/description.md:5]: nope

**Review by gauravrajguru — DISMISSED** [review]

**Review by gauravrajguru — APPROVED** [review]

**Review by MadhuM02 — APPROVED** [review]

**Pull request merged**